e2e/windows/scenarios/windows/property-windows-template.json

```diff
@@ -3,7 +3,7 @@
     "properties": {
       "orchestratorProfile": {
         "orchestratorType": "Kubernetes",
-        "orchestratorVersion": "1.23.12",
+        "orchestratorVersion": "${orchestratorVersion}",
```

Comment:
Two questions here:
1. When we set "WindowsPackageURL" manually, the orchestractorVersion may be wrong here?
2. Have you checked the kubernetes version on Azure Portal when use hotfix package url?

Reply:
1. Based on my experience, the kubernetes version on Azure Portal  (or `kubectl get nodes`) comes from the version in the windows package url. Not sure if this `orchestratorVersion` is needed.
2. No I haven't yet. I only checked the versions for non-hotfix which are the same as that in the package url even if the `orchestratorVersion` is different. Will double check on this.

If this is not necessary proven by experiments I think I can just remove this field? @junjiezhang1997

Reply:
For the pipeline using 1.22.1 hotfix, the version returned by `kubectl get nodes` is 1.22.1 as seen in the pipeline logs.
Same for 1.25.2.

Reply:
Another problem needs to be considered is that your resource group name is named by KUBERNETES _VERSION, so maybe you need to clarify the resource group name with the hotfix name to avoid ambiguity?

Reply:
Summary on the discussion with @junjiezhang1997 
- the deployment node that joined the test-cluster is expected to have k8s version equals to whatever comes from the `windowsPackageURL`.
- the resource group name contains this windows package version and an optional "hotfix" to disambiguate. $e.g.$ testing `1.25.2-hotfix` the RG ends with `1252-hotfix`.
- the k8s version of the linux `test-cluster` is `KUBERNETES_VERSION` - a pipeline UI variable added by @junjiezhang1997 and it defaults to `1.24.9`.